doc/tunnel/SAI-Proposal-ImprovedP2PTunnels.md

```diff
@@ -0,0 +1,61 @@
+# Introduction
+As of SAI version 1.5 the “tunnel” has a p2mp connotation. It holds the VTEP SIP whereas there is no DIP. The DIP is specified as part of the FDB entry or as part of Next Hop entry.
```

Comment:
How can P2P tunnel model be applied to Next Hop entry?

Reply:
The SAI nexthop can refer to the p2p tunnel id, tunnel vni and MAC just as it does today for p2mp. 
However it is not the expectation to replace the p2mp tunnel. It is expected that nexthop programming and FDB continue to use the p2mp version.